Dockerfile

```diff
@@ -1,19 +1,24 @@
-ARG VERSION=v0.400.3
 ARG LUET_VERSION=0.35.5
+ARG LEAP_VERSION=15.5
 
 FROM quay.io/luet/base:$LUET_VERSION AS luet
 
 FROM golang AS builder
-ARG BINARY_VERSION=v0.0.0
+ARG VERSION=v0.0.0
 WORKDIR /work
 ADD go.mod .
 ADD go.sum .
 RUN go mod download
 ADD . .
-RUN CGO_ENABLED=0 go build -ldflags "-X main.version=${BINARY_VERSION}" -o auroraboot
-
-FROM quay.io/kairos/osbuilder-tools:$VERSION
+ENV CGO_ENABLED=0
+ENV VERSION=$VERSION
+RUN go build -buildvcs=false -ldflags "-X main.version=${VERSION}" -o auroraboot
 
+FROM opensuse/leap:$LEAP_VERSION AS default
+RUN zypper ref && zypper dup -y
+## ISO+ Arm image + Netboot + cloud images Build depedencies
+RUN zypper ref && zypper in -y bc qemu qemu-tools jq cdrtools docker git curl gptfdisk kpartx sudo xfsprogs parted binutils \
```

Comment:
can we add a TODO here to track grub artifacts here and below? We may not need some of them anymore with aurora bundling the iso cd images